tools/clang/include/clang/SPIRV/SpirvBuilder.h

```diff
@@ -213,6 +213,9 @@ class SpirvBuilder {
   SpirvUnaryOp *createUnaryOp(spv::Op op, QualType resultType,
                               SpirvInstruction *operand, SourceLocation loc);
 
```

Comment:
nit: Remove newline to match surrounding formatting

Reply:
Done.